frontend/javascripts/oxalis/model/accessors/dataset_accessor.ts

```diff
@@ -341,6 +341,12 @@ export function determineAllowedModes(settings?: Settings): {
     allowedModes,
   };
 }
+
+export function getMaximumSegmentIdForLayer(dataset: APIDataset, layerName: string) {
+  const valueRange = getDefaultIntensityRangeOfLayer(dataset, layerName);
+  return valueRange[1];
+}
+
```

Comment:
@philippotto To me it looks like I successfully renamed it. Maybe you overlooked the `outdated` above the code lines where you suggested to give the result of `getDefaultIntensityRangeOfLayer(dataset, layerName);` a name?

Reply:
I think, we are talking about different things :sweat_smile: My suggestion is to rename the function `getDefaultIntensityRangeOfLayer` to `getDefaultValueRangeOfLayer`. That's what I tried to express in https://github.com/scalableminds/webknossos/pull/7240#discussion_r1283122533 where I quoted you asking whether this should be done. You probably interpreted my "this is what I meant" so that I confirmed that changes you did instead of your quote?

Reply:
> You probably interpreted my "this is what I meant" so that I confirmed that changes you did instead of your quote?

Yes indeed :see_no_evil: xD, sorry for the confusion

Reply:
Is there anything else you noticed or is this pr ready to :shipit: ?